src/components/Modal/OpenOrFundChannelModal.tsx

```diff
@@ -90,7 +99,13 @@ export const OpenChannelModal = ({
 
   return (
     <>
-      <button onClick={handleOpenChannelDialog}>{modalBtnText ? modalBtnText : 'Open Channel'}</button>
+      <IconButton
+        iconComponent={
+          type === 'open' ? <HubIcon /> : type === 'fund' ? <MonetizationOnIcon /> : <MonetizationOnIcon />
+        }
```

Comment:
Can this be change to this?:

```suggestion
        iconComponent={
          type === 'open' ? <HubIcon /> : <MonetizationOnIcon />
        }
```

I'm open to discussion since even if it's less verbose it might also be less readable to some extent

Reply:
changed to a switch